README.md

```diff
@@ -23,21 +23,73 @@
 
 ## Introduction
 
-Jujutsu is a
-[Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility)
-[DVCS](https://en.wikipedia.org/wiki/Distributed_version_control). It combines
-features from Git (data model,
-[speed](https://github.com/martinvonz/jj/discussions/49)), Mercurial (anonymous
-branching, simple CLI [free from "the
-index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
-[revsets](https://martinvonz.github.io/jj/latest/revsets), powerful
-history-rewriting), and Pijul/Darcs ([first-class
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
-([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
-[undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
-system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
+Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
+system for software projects. You use it to get a copy of the code, track
+changes to the code, and finally publish those changes for others to see and
+use. Jujutsu is unlike most other systems, because internally it abstracts
+the data model used to store your content from its user facing algorithms and
+interface. This allows it to serve as a version control interface with many
+possible physical backends, such as Git (today), while also open to other
+potential backends, like Mercurial, or even more centralized systems like
+Perforce.
+
+Today, by default, we use the Git data model to store and track content, so
+it's widely compatible with your favorite Git-based tools. All core Jujutsu
+developers use Jujutsu to develop it, right here on GitHub! But it will also
+work with your favorite Git forges, too.
+
+Jujutsu combines many distinct design features from other version control
+systems into a single user interface. Those other systems include:
+
+- **Git**: Like Git, we make an effort to [be fast][perf], from a snappy
+  UX to efficient algorithms. It also by default shares Git's data model, for
+  wide interoperability and ease of onboarding.
+
+- **Mercurial & Sapling**: Many Mercurial inspired features include the [revset]
+  language to select commits. There is [no explicit index][no-index] or "staging
+  area". Branches are "anonymous" like Mercurial, so you don't need to make up
+  a name for each small change. Primitives for rewriting history are powerful
+  and simple.
+
+- **Pijul & Darcs**: Jujutsu keeps track of conflicts as [first-class objects][conflicts]
```

Comment:
Not action needed: They're not actually stored as conflict objects anymore (I need to update that doc). These days they are stored a multiple trees linked from a single commit. It's probably still okay to call them "first-class objects" here even though they're not objects in the same way as commits, trees, and blobs are.